### PR TITLE
Adjust newly built building appearance and outline blinking speed

### PR DESCRIPTION
**Pull request by dydzio0614**
Make buildings appearance look OK with new framerate + match timings description from comments

### CODE REVIEW

**Comment by IvanSavenko** [comment]
Why not use time instead of frame count?
Change those constants to milliseconds and instead of incrementing counter by 1 on each update increment it by time delta (from framerate manager)

**Comment by dydzio0614** [comment]
I dont wanna break anything for now, just minor tuning for next potential stream testing

**Comment by IvanSavenko** [comment]
Judging from code - such changes would be small, contained & unlikely to have side effects

**Review by IvanSavenko — APPROVED** [review]

File: client/windows/CCastleInterface.cpp [hunk 1]
```diff
@@ -632,9 +632,9 @@ void CCastleBuildings::addBuilding(BuildingID building)
 		{
 			//reset animation
 			if(structures.size() == 1)
-				buildingRect->stateCounter = 0; // transparency -> fully visible stage
+				buildingRect->stateTimeCounter = 0; // transparency -> fully visible stage
 			else
-				buildingRect->stateCounter = 16; // already in fully visible stage
+				buildingRect->stateTimeCounter = 500; // already in fully visible stage
```
Comment (Nordsoft91) [inline client/windows/CCastleInterface.cpp:637]: 1. Move S1_TRANSP and other consts outside of class method
2. 
```suggestion
				buildingRect->stateTimeCounter = S1_TRANSP; // already in fully visible stage
```

File: client/windows/CCastleInterface.h [hunk 2]
```diff
@@ -42,6 +42,14 @@ class CBuildingRect : public CShowableAnim
 {
 	std::string getSubtitle();
 public:
+	enum EBuildingCreationAnimationPhases : uint32_t
```
Comment (Nordsoft91) [inline client/windows/CCastleInterface.h:45]: minor suggestion - make it private

**Review by Nordsoft91 — APPROVED** [review]

File: client/windows/CCastleInterface.h (hunk 2 above)
Comment (dydzio0614) [reply client/windows/CCastleInterface.h:45]: it wont work because another class uses it

**Pull request merged**